README.md

```diff
@@ -91,6 +91,61 @@ sudo apt-get install protobuf-compiler libprotoc-dev
 pip install onnx
 ```
 
+### Windows
+When building on Windows it is highly recommended that you also build protobuf locally as a static library. The version distributed with conda-forge is a DLL and this is a conflict as ONNX expects it to be a static lib.
+
+#### Instructions to build protobuf and ONNX on windows
+Step 1 : Build protobuf locally
+```
+git clone https://github.com/protocolbuffers/protobuf.git
+cd protobuf
+git checkout 3.9.x
+cd cmake
+# Explicitly set -Dprotobuf_MSVC_STATIC_RUNTIME=OFF to make sure protobuf does not statically link to runtime library
+cmake -G "Visual Studio 15 2017 Win64" -Dprotobuf_MSVC_STATIC_RUNTIME=OFF -Dprotobuf_BUILD_TESTS=OFF -Dprotobuf_BUILD_EXAMPLES=OFF -DCMAKE_INSTALL_PREFIX=<protobuf_install_dir>
+msbuild protobuf.sln /m /p:Configuration=Release
+msbuild INSTALL.vcxproj /p:Configuration=Release
+```
+
+Step 2: Build ONNX
+```
+# Get ONNX
+git clone https://github.com/onnx/onnx.git
+cd onnx
+git submodule update --init --recursive
+
+# Set environment variables to find protobuf and turn off static linking of ONNX to runtime library.
+# even better option is to add it to user\system PATH so this step can be performed only once.
+# For more details check https://docs.microsoft.com/en-us/cpp/build/reference/md-mt-ld-use-run-time-library?view=vs-2017
+set PATH=<protobuf_install_dir>\bin;%PATH%
+set USE_MSVC_STATIC_RUNTIME=0
+
+# Build ONNX
+python setup.py install
+```
+
+If you do not want to build protobuf and instead want to use protobuf from conda forge then follow these instructions. 
```

Comment:
Are there any requirements or recommendations for python version?

Reply:
In CIs - for Linux we test using python 2.7 and 3.6 , on Mac we test using 3.7 and 3.7 whereas on in windows ci we test with python 3.5, 3.6, 3.7

Right now I do not want to add any recommendations without understanding what is our official stance. There is no other documentation which lists the officially supported versions...